diff -ru chan_capi-1.1.6.orig/chan_capi_devstate.c chan_capi-1.1.6.patched/chan_capi_devstate.c
--- chan_capi-1.1.6.orig/chan_capi_devstate.c	2011-10-01 18:32:26.000000000 +0200
+++ chan_capi-1.1.6.patched/chan_capi_devstate.c	2015-10-05 19:11:01.000000000 +0200
@@ -113,7 +113,7 @@
 {
 	if (capiChatProviderRegistered != 0) {
 #ifdef CC_AST_HAS_VERSION_1_6
-		ast_devstate_changed((inUse != 0) ? AST_DEVICE_INUSE : AST_DEVICE_NOT_INUSE, "capichat:%s", roomName);
+		ast_devstate_changed((inUse != 0) ? AST_DEVICE_INUSE : AST_DEVICE_NOT_INUSE, AST_DEVSTATE_CACHABLE, "capichat:%s", roomName);
 #else
 		ast_device_state_changed("capichat:%s", roomName);
 #endif
@@ -130,7 +130,7 @@
 			((capiController->nfreebchannels >= capiController->nfreebchannelsHardThr) &&
 				(capiController->nfreebchannels - channelsChanged < capiController->nfreebchannelsHardThr))) {
 #ifdef CC_AST_HAS_VERSION_1_6
-		ast_devstate_changed(AST_DEVICE_UNKNOWN, CC_MESSAGE_BIGNAME"/I%d/congestion", capiController->controller);
+		ast_devstate_changed(AST_DEVICE_UNKNOWN, AST_DEVSTATE_CACHABLE, CC_MESSAGE_BIGNAME"/I%d/congestion", capiController->controller);
 #else
 		ast_device_state_changed (CC_MESSAGE_BIGNAME"/I%d/congestion", capiController->controller);
 #endif
